BuildROS2/pre_build_ros2_base.sh

```diff
@@ -15,66 +15,66 @@ Install ROS2 $ROS2_DISTRO
+# #locale  # verify settings
 
 
-## Add the ROS 2 apt repository
-apt-cache policy | grep universe
+# ## Add the ROS 2 apt repository
+# apt-cache policy | grep universe
 
-sudo apt install software-properties-common
-sudo add-apt-repository universe
+# sudo apt install software-properties-common
+# sudo add-apt-repository universe
 
-sudo apt update && sudo apt install curl gnupg2 lsb-release
-sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key  -o /usr/share/keyrings/ros-archive-keyring.gpg
+# sudo apt update && sudo apt install curl gnupg2 lsb-release
+# sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key  -o /usr/share/keyrings/ros-archive-keyring.gpg
 
-echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(source /etc/os-release && echo $UBUNTU_CODENAME) main" | sudo tee /etc/apt/sources.list.d/ros2.list > /dev/null
+# echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(source /etc/os-release && echo $UBUNTU_CODENAME) main" | sudo tee /etc/apt/sources.list.d/ros2.list > /dev/null
 
 ## Install development tools and ROS tools
-sudo apt update && sudo apt install -y \
-  build-essential \
-  cmake \
-  git \
-  libbullet-dev \
-  python3-colcon-common-extensions \
-  python3-flake8 \
-  python3-pip \
-  python3-pytest-cov \
-  python3-rosdep \
-  python3-setuptools \
-  python3-vcstool \
-  wget
-# install some pip packages needed for testing
-python3 -m pip install -U \
-  argcomplete \
-  flake8-blind-except \
-  flake8-builtins \
-  flake8-class-newline \
-  flake8-comprehensions \
-  flake8-deprecated \
-  flake8-docstrings \
-  flake8-import-order \
-  flake8-quotes \
-  pytest-repeat \
-  pytest-rerunfailures \
-  pytest
-# install Fast-RTPS dependencies
-sudo apt install --no-install-recommends -y \
-  libasio-dev \
-  libtinyxml2-dev
-# install Cyclone DDS dependencies
-sudo apt install --no-install-recommends -y \
-  libcunit1-dev
+# sudo apt update && sudo apt install -y \
+#   build-essential \
+#   cmake \
+#   git \
+#   libbullet-dev \
+#   python3-colcon-common-extensions \
+#   python3-flake8 \
+#   python3-pip \
+#   python3-pytest-cov \
+#   python3-rosdep \
+#   python3-setuptools \
+#   python3-vcstool \
+#   wget
+# # install some pip packages needed for testing
+# python3 -m pip install -U \
+#   argcomplete \
+#   flake8-blind-except \
+#   flake8-builtins \
+#   flake8-class-newline \
+#   flake8-comprehensions \
+#   flake8-deprecated \
+#   flake8-docstrings \
+#   flake8-import-order \
+#   flake8-quotes \
+#   pytest-repeat \
+#   pytest-rerunfailures \
+#   pytest
+# # install Fast-RTPS dependencies
+# sudo apt install --no-install-recommends -y \
+#   libasio-dev \
+#   libtinyxml2-dev
+# # install Cyclone DDS dependencies
+# sudo apt install --no-install-recommends -y \
+#   libcunit1-dev
```

Comment:
I assume this script is largely commented out because you've moved much of the software installation/configuration to the Dockerfile. @yuokamoto do we use this script in any internal Docker builds? If so a better option might be to split this script out into stuff that gets installed and stuff that gets built.

Reply:
I mostly made the the bash script useless for building the docker environment. The commented part were just for testing but since there is a docker image now, there is no need to keep the bash files really.